README.md

```diff
@@ -22,12 +20,10 @@ How?
 server.register([
   register: require('ot-hapi-queryparams'),
   options: {
-  delete_original: true/false // remove the origin query param attribute
-  //ONLY ONE OPTION SHOULD BE SET TO TRUE
-    camel_case: true, // user_name => userName , UserName => userName
-    all_caps: true,   // username => USERNAME
-    lower_case: true, // UserName => username
-    first_cap: true, // userName => UserName
+  deleteOriginal: true/false // remove the origin query param attribute
+  //only one of these should be set at a time.
+    schemaCamelCase: { userName: Joi.string().required() }, // username => userName (given a schema with userName as a key)
```

Comment:
I would just call this schemaCase, instead of camel case (since it just matches what it was passed in)

Reply:
or camel case. Please look at my overall review comments